doc/source/security/encrypting-undo-tablespace.rst

```diff
@@ -0,0 +1,43 @@
+.. _undo-tablespace-encryption:
+
+================================================================================
+Encrypting the Undo Tablespace
+================================================================================
+
+The undo data may contain sensitive information about the database operations.
+You can encrypt the data in an undo log using the :variable:
+`innodb_undo_log_encrypt` option. You can change the setting for this variable
+in the configuration file. The undo data encryption must be enabled; the feature
```

Comment:
or as a startup parameter, or during runtime as a global variable... it's not limited to the config file.

Reply:
added.